config/passport.js

```diff
@@ -48,7 +50,7 @@ module.exports = function(passport) {
           user.google.avatar = profile.image.url.split('?')[0]
 
           user.save(function(err) {
-            if (err) throw err
+            if (err) console.log('Failed to register new user')
```

Comment:
The user will continue unauthenticated, and we will have the error logged. It works for the moment and I will have a bug filled for this. :) thx